specs/yara_events.table

```diff
@@ -4,13 +4,15 @@ schema([
     Column("target_path", TEXT, "The path scanned"),
     Column("category", TEXT, "The category of the file"),
     Column("action", TEXT, "Change action (UPDATE, REMOVE, etc)"),
-    Column("transaction_id", BIGINT, "ID used during bulk update"),
     Column("matches", TEXT, "List of YARA matches"),
     Column("count", INTEGER, "Number of YARA matches"),
     Column("strings", TEXT, "Matching strings"),
     Column("tags", TEXT, "Matching tags"),
     Column("time", BIGINT, "Time of the scan"),
     Column("eid", TEXT, "Event ID", hidden=True),
 ])
+extended_schema(DARWIN, [
+    Column("transaction_id", BIGINT, "ID used during bulk update"),
+])
```

Comment:
This PR didn't make any changes to this column, but I noticed that the code comments indicate it is only ever returned on macOS.